src/pages/app/module_2/styles/QuotaManagementStyles.css

```diff
@@ -2,7 +2,7 @@
 .quota-page{
     background-color:var(--global-background-color);
     display:grid;
-    width: 1600px;
+    width: 1535px;
```

Comment:
One of the convention for styling container component is not to use **customized width** like this. It should be controlled by the **display** property to make the component stretch out the whole dimension (in this case is the width of the screen)

Reply:
Ok I will check it when I go home
@Dev01-vinamr-Ken